### ISSUE TITLE
[1.12.2] [Request] Option To Turn Malasis Door Integration Off

### ISSUE THREAD

**Issue by Sanofus** [AbrarSyed/SecretRoomsMod-forge#252]
I noticed that SRM adds compat to MD but I do not want to have this integration because it causes light issues. The SRM does not add any config to prevent compat. Could this be added?

**Comment by Wyn-Price** [comment]
What are the light issues, I might be able to fix those as well

**Comment by Sanofus** [comment]
Yeah I decided to turn off all animation of doors because not all doors in my modpack have this implemented. However I made you screenshots of the light issue. First with and second without Optifine but it is the same result.

This happens after I added MD.
![2018-06-09_21 52 21](https://user-images.githubusercontent.com/26455012/41195682-b8212c62-6c32-11e8-9534-843218e21077.png)
![2018-06-09_22 01 42](https://user-images.githubusercontent.com/26455012/41195684-c17b4cfc-6c32-11e8-8a05-52d47b40ca93.png)


There is also another issue that occured before. I do not know if this is already known. This happens before I added MD.
![2018-06-09_21 52 29](https://user-images.githubusercontent.com/26455012/41195696-e2bab4fc-6c32-11e8-8677-a92e54301acb.png)
![2018-06-09_22 01 52](https://user-images.githubusercontent.com/26455012/41195699-ea62d84c-6c32-11e8-8dc8-d288713dbcd0.png)

There also a third issue. As you can see the door imitates black granite but in HWYLA it is declared as red granite. That might be caused by meta data that is not used for gathering the block variant.

**Comment by Wyn-Price** [comment]
I'll look into the hwyla stuff, but the inside of the door is always going to be dark, due to the way the light engine works

**Comment by Wyn-Price** [comment]
But the outside of the door shouldn't be like that no

**Comment by Sanofus** [comment]
yeah. it is all black granite.

And I thought the inner was too dark because you cannot see which block it is. But this might not be an issue. Secret Doors won't be opn for long time.

**Comment by Wyn-Price** [comment]
Is the block texture correct?

**Comment by Sanofus** [comment]
The block underneath the door? yes.

**Comment by Wyn-Price** [comment]
Can you take the screenshots again but with better light

**Comment by Wyn-Price** [comment]
Like light it up with torches or something

**Comment by Sanofus** [comment]
It seems that it only happen if a block is placed above the door.

![2018-06-09_22 55 03](https://user-images.githubusercontent.com/26455012/41196021-5082b3ae-6c38-11e8-81cb-1d1894e0317b.png)
![2018-06-09_22 55 10](https://user-images.githubusercontent.com/26455012/41196022-52d1c168-6c38-11e8-9683-1d85ac33ec9b.png)

**Comment by diademos** [comment]
Hey, i have the same. any updates on that?
Regards
Diademos

**Comment by diademos** [comment]
![2018-08-19_11 52 12](https://user-images.githubusercontent.com/35638323/44307658-60e02100-a3a6-11e8-835d-418d69fbbed3.png)
This is how it looks like on the the Server

**Comment by Wyn-Price** [comment]
Im looking into it
